src/test/java/no/unit/nva/cristin/projects/FetchOneCristinProjectTest.java

```diff
@@ -109,15 +110,15 @@ void handlerReturnsBadGatewayWhenBackendThrowsIoException() throws Exception {
     }
 
     @Test
-    void handlerReturnsServerErrorExceptionWhenBackendThrowsUriSyntaxException() throws Exception {
+    void handlerReturnsServerErrorExceptionWhenBackendThrowsGenericException() throws Exception {
         cristinApiClientStub = spy(cristinApiClientStub);
 
-        doThrow(URISyntaxException.class).when(cristinApiClientStub).getProject(any(), any());
+        doThrow(RuntimeException.class).when(cristinApiClientStub).getProject(any(), any());
         handler = new FetchOneCristinProject(cristinApiClientStub, environment);
-        GatewayResponse<NvaProject> nextResponse = sendQueryWithId(DEFAULT_ID);
+        GatewayResponse<NvaProject> response = sendQueryWithId(DEFAULT_ID);
```

Comment:
This is a GatewayResponse<Problem>  I think

Reply:
you mean i should call response for gatewayResponse?

Reply:
Sorry I mean Problem class, should be fixed in subsequent PR